fctesting/utils.go

```diff
@@ -43,3 +46,24 @@ func RequiresRoot(t testing.TB) {
 			rootDisableEnvName)
 	}
 }
+
+func newLogger() *log.Logger {
+	str := os.Getenv(logLevelEnvName)
+	if str != "" {
+		logLevel, err := log.ParseLevel(str)
+		if err != nil {
+			panic(err)
+		}
+		return &log.Logger{
+			Out:   os.Stdout,
+			Level: logLevel,
+		}
+	} else {
```

Comment:
Don't need else can just return on parent's scope.

Reply:
Right and golint also pointed out that.